umbra-js/test/Umbra.test.ts

```diff
@@ -1,20 +1,23 @@
 import { ethers } from 'hardhat';
 import hardhatConfig from '../hardhat.config';
 import { Umbra } from '../src/classes/Umbra';
-import { BigNumberish, BigNumber, StaticJsonRpcProvider, Wallet } from '../src/ethers';
+import { BigNumberish, BigNumber, StaticJsonRpcProvider, Wallet, ContractTransaction } from '../src/ethers';
 import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
 import { HardhatNetworkHDAccountsUserConfig } from 'hardhat/src/types/config';
 import { expect } from 'chai';
 import { expectRejection } from './utils';
 import { testPrivateKeys } from './testPrivateKeys';
-import type { ChainConfig } from '../src/types';
+import type { ChainConfig, SendBatch, UserAnnouncement } from '../src/types';
 import {
   TestToken as ERC20,
   Umbra as UmbraContract,
   TestTokenFactory as ERC20__factory,
   UmbraFactory as Umbra__factory,
 } from '@umbra/contracts-core/typechain';
-
+import { parseOverrides, assertSufficientBalance, assertValidStealthAddress } from '../src/classes/Umbra';
+import { UMBRA_BATCH_SEND_ABI } from '../src/utils/constants';
+import { KeyPair } from '../src';
+import { blockedStealthAddresses } from '../src/utils/utils';
 const { parseEther } = ethers.utils;
```

Comment:
Nit, let's separate imports from variables with a line break
```suggestion
import { blockedStealthAddresses } from '../src/utils/utils';

const { parseEther } = ethers.utils;
```